src/scheduler/node_info.c

```diff
@@ -2244,24 +2244,24 @@ eval_placement(status *policy, selspec *spec, node_info **ninfo_arr, place *pl,
 	resource_resv *resresv, unsigned int flags,
 	nspec ***nspec_arr, schd_error *err)
 {
-	np_cache *npc;
-	node_partition **hostsets = NULL;
-	char *host_arr[2] = {"host", NULL};
-	int i;
-	int k;
-	int tot = 0;
-	int c = -1;
-	int cur_flt_lic;
-	nspec **nsa = NULL;
-	nspec **ns_head = NULL;
-	char logbuf[MAX_LOG_SIZE];
-	char reason[MAX_LOG_SIZE];
-	resource_req *req;
-	schd_resource *res;
-	selspec *dselspec = NULL;
-	int do_exclhost = 0;
-	node_info **nptr = NULL;
-	static schd_error *failerr = NULL;
+	np_cache		*npc = NULL;
+	node_partition		**hostsets = NULL;
+	char			*host_arr[2] = {"host", NULL};
+	int			i = 0;
+	int			k = 0;
+	int			tot = 0;
+	int			c = -1;
+	int			cur_flt_lic = resresv->server->flt_lic;
```

Comment:
This will crash if resresv or resresv->server is NULL.  Assign this after you've already checked reseresv for NULL.

Reply:
done!